bin/pycbc_compress_bank

```diff
@@ -157,6 +156,8 @@ for ii in range(imin, imax):
     htilde = bank[ii-imin]
     tmplt = bank.table[ii-imin]
     fmin=tmplt.f_lower
+    template_duration=htilde.chirp_length
```

Comment:
I don't think this will work; `chirp_length` will just be None here (look at `FilterBank.__getitem__`). You can get the duration of the waveform from the difference in the sample frequencies of the first two compression points. So, move this down to after the compressed waveform is created. Then do:
```
template_duration = 1./(hcompressed.sample_points[1] - hcompressed.sample_points[0])
```
Then save to the output table.